src/TestSuite/ConsoleIntegrationTestTrait.php

```diff
@@ -66,21 +66,21 @@ trait ConsoleIntegrationTestTrait
     /**
      * Console output stub
      *
-     * @var \Cake\TestSuite\Stub\ConsoleOutput|\PHPUnit\Framework\MockObject\MockObject|null
+     * @var \Cake\TestSuite\Stub\ConsoleOutput
      */
     protected $_out;
 
     /**
      * Console error output stub
      *
-     * @var \Cake\TestSuite\Stub\ConsoleOutput|\PHPUnit\Framework\MockObject\MockObject|null
+     * @var \Cake\TestSuite\Stub\ConsoleOutput
      */
     protected $_err;
 
     /**
      * Console input mock
      *
-     * @var \Cake\Console\ConsoleInput|\PHPUnit\Framework\MockObject\MockObject|null
```

Comment:
Weren't these added to fix an IDE? Are these still needed?

Reply:
The `exec()` method set's the property to concrete class `\Cake\Console\ConsoleInput` so the `|\PHPUnit\Framework\MockObject\MockObject` isn't necessary.